src/importexport/musicxml/internal/musicxml/musicxmlpart.h

```diff
@@ -19,33 +19,15 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
+#ifndef MUSICXMLPART_H
+#define MUSICXMLPART_H
```

Comment:
Since recently, we can replace these `#ifndef/#define/#endif`guards with only `#pragma once` at the beginning of a header file